pkg/util/utils_supported.go

```diff
@@ -14,6 +14,17 @@ import (
 	"syscall"
 )
 
+const _CGROUP2_SUPER_MAGIC = 0x63677270
+
+// IsCgroup2UnifiedMode returns whether we are running in cgroup 2 unified mode.
+func IsCgroup2UnifiedMode() (bool, error) {
```

Comment:
Can you put this into buildah rather then into Podman so both tools can share the check?

Reply:
we are not currently doing any the same check in Buildah as we do for Podman when resources are specified.